### ISSUE TITLE
[taskbar] add a on_hover_drag event

### ISSUE THREAD

**Issue by Arnaudv6** [Alexays/Waybar#866]
Thanks for this awesome bar + module.

The idea is you're dragging an item from a window, and want to drop it on an other window, which is either minimized or buried under windows.
after one sec hovering the destination window's item on taskbar, said window gets (un-minimized) and raised so we can drop on it.

**Comment by Arnaudv6** [comment]
I can't think of an other usage for this event : maybe it can be non-user-facing, hard-coded behavior

**Comment by glyh** [comment]
I also support this, since I found the default hover effect make the icon unreadable for workspaces tag in sway.

**Comment by Arnaudv6** [comment]
I do not recognize my feature request in the problem you describe, @glyh

**Comment by mazunki** [comment]
I am not exactly sure what this issue is pointing to, but it seems related to something I'd love to see: When moving a window in Sway, I often want to simply "drop it into another workspace", without actually going to that workspace.

Current workaround involves moving it temporarily to another monitor, or using keybindings.

Picture related, 15 Firefox windows opened from a restored session, in same workspace:
![image](https://user-images.githubusercontent.com/32819373/218272478-9c369ff7-65aa-4289-8a6f-3791cdc96757.png)

If issue is pointing to something else I can open an unrelated issue.

**Comment by Arnaudv6** [comment]
@mazunki, you'd like to drop windows to workplaces, right?
Whereas I would like to raise windows when I hover their taskbar button while dragging something.

**Comment by mazunki** [comment]
@Arnaudv6 Yeah, that's correct. Preview-on-hover could be useful for drag-and-drop, I suppose. Change-on-hover would probably work for my use case, but I'd prefer preview-on-hover tbh.
